src/python/BuildingControlsSimulator/ControlModels/Deadband.py

```diff
@@ -1,78 +1,140 @@
 # created by Tom Stesco tom.s@ecobee.com
 
 import attr
+import pandas as pd
+import numpy as np
 
 from BuildingControlsSimulator.ControlModels.ControlModel import ControlModel
-from BuildingControlsSimulator.ControlModels.ControlModel import HVAC_modes
+from BuildingControlsSimulator.DataClients.DataStates import STATES
+from BuildingControlsSimulator.DataClients.DataSpec import Internal
+from BuildingControlsSimulator.Conversions.Conversions import Conversions
 
 
 @attr.s
 class Deadband(ControlModel):
-    """Deadband controller
+    """Deadband controller"""
 
-    Example:
-    ```python
-    from BuildingControlsSimulator.ControlModels.Deadband import Deadband
-    ```
+    deadband = attr.ib(default=1.0)
+    input_states = attr.ib(
+        default=[
+            STATES.THERMOSTAT_TEMPERATURE,
+            STATES.TEMPERATURE_STP_COOL,
+            STATES.TEMPERATURE_STP_HEAT,
+        ]
+    )
 
-    """
+    output_states = attr.ib(
+        default=[
+            STATES.TEMPERATURE_CTRL,
+            STATES.AUXHEAT1,
+            STATES.AUXHEAT2,
```

Comment:
I'm a bit surprised that all of the fields show up here for the deadband considering it only touches a more limited set. Feel free to ignore this remark.

Reply:
The output states are somewhat standardized and though this simple example controller doesnt do multistage actuation the output still should support it by always setting it to off.